modules/yugabytedb/yugabytedb.go

```diff
@@ -0,0 +1,249 @@
+package yugabytedb
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+	"github.com/yugabyte/gocql"
```

Comment:
Nit: can you verify golangci-lint is not complaining about the order of the imports? I have a feeling that it will 🤞